vagrant-cfg/common.yaml

```diff
@@ -0,0 +1,196 @@
+---
+
+## common config variables
+path: /var/www/cmf
+vhost: cmf.lo
+
+## vagrant config
+box_name: mayflower/trusty64-puppet3    # or e.g. mayflower/centos-7.0-64-puppet3
+
+#box_cpus: 2
+#box_memory: 1024
+
+# in order to use the production mode at the cmf, please uncomment the following line.
+#production: true
+
+ip: 172.22.22.22
+nfs: true
+#proxy: http://proxy-server:8080/
+#no-proxy: localhost,127.0.0.1
+
+## puppet profiles
+profile::packages::names:
+  - vim
+  - bash-completion
+  - wget
+  - curl
+
+profile::database::mysql: true
+profile::database::postgresql: false
+profile::database::mongodb: false
+profile::database::redis: false
+profile::database::oracle_instantclient: false
+
+profile::webserver::nginx: true
+profile::webserver::apache: false
+
+profile::interpreter::php: true
+profile::interpreter::hhvm: false
+
+profile::javascript::nodejs: false
+profile::javascript::less: false
+
+profile::frontend::compass: false
+profile::frontend::bower: false
+profile::frontend::coffee_script: false
+profile::frontend::grunt: false
+
+profile::app::symfony2: true
+profile::app::yii1: false
+profile::app::yii2: false
+profile::app::zend_framework1: false
+profile::app::slim: false
+profile::app::php_vhost: false  # simple vhost that sends all *.php to php-fpm
+profile::app::standalone_app: false  # proxy to an app that comes with a webserver (rails, django, flask)
+
+profile::queue::rabbitmq: false
+
+## puppet files and directories sync
+# profile::sync::files:
+#   /home/vagrant/.profile:
+#     source: file:///vagrant/.profile
+# profile::sync::directories:
+#   /home/vagrant/bin:
+#     source: file:///vagrant/bin
+
+
+## puppet modules
```

Comment:
really, puppet? this is a religious war... but if it works well here i guess we can use that.